products/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+# from order.models import Order 
+# from account.models import Account
+
+class Main_category(models.Model):
+    name                 = models.CharField(max_length=50)
+    class Meta:
+        db_table ='main_categories'
```

Comment:
```suggestion
class Main_category(models.Model):
    name = models.CharField(max_length=50)

        class Meta:
        db_table ='main_categories'
```
내부 클래스 선언하실 때, 한 빈 줄 한 줄 넣어주세요. 아래에 있는 모든 클래스 전부 공통으로 수정해주세요.
그리고 `=`을 기준으로 정렬할 때는 그 클래스 안에 있는 속성끼리만 맞추시면 됩니다. 전체 파일에서 모두 맞추시는 것은 오히려 가독성이 더 떨어집니다 :) 수정해주세요